content/en/digital_certificate.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Digital Certificate
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+Digital certificates or public key certificates are issued by a certificate authority (CA). These certificates are electronic files or passwords that allow entities to exchange data or resources over the internet securely. 
+Certificates authenticate the user through encryption methods and a set of roles, policies, and resources used to manage, use and revoke certificates and encryptions. Just as a passport certifies an individual's identity, a digital certificate establishes the user's identity. Note that a user can be a person or service within a [microservice](https://glossary.cncf.io/microservices/) app. 
+
+## Problem it addresses
+Digital certificates help solve web-based communication security issues. Networks were developed without much thought to security. The most generic set of rules and policies to process data known as protocols, were designed around precepts of robust communication rather than secure transactions. No centralized mechanism was designed and implemented to ensure security over network communiications. Consequently each application developed its own negotiation strategy. Rather than building a generic, reusable security mechanism, applications relied on proprietary or embedded techniques for shuffling account information across the network to the platform or application for validation.
```

Comment:
```suggestion
Networks, such as the internet, are inherently insecure. Communication is managed through protocols (rules and policies to process data), but these are generic and designed to focus on robust communication rather than secure transactions. There is no centralized mechanism to secure communication. Consequently, each application must have its authentication strategy. Rather than building a generic, reusable security mechanism, applications relied on proprietary or embedded techniques for shuffling account information across the network for validation.
```

Reply:
I removed the first sentence because, ideally, in the problem it addresses section, we don't even mention the solution. The remaining changes are stylistic.

Reply:
@CathPag thankyou for the feedback. I am fixing it right away